bootstrap/sql/com.mysql.cj.jdbc.Driver/v006__create_db_connection_info.sql

```diff
@@ -13,3 +13,12 @@ CREATE TABLE IF NOT EXISTS web_analytic_event (
 
 UPDATE bot_entity
 SET json = JSON_INSERT(JSON_REMOVE(json, '$.botType'), '$.provider', 'system');
+
+
+UPDATE pipeline_service_entity 
+SET json = JSON_INSERT(json ,'$.connection.config.configSource.hostPort', '$.connection.config.hostPort'),
+WHERE serviceType = 'Dagster'
+
+UPDATE pipeline_service_entity 
+SET json = JSON_REMOVE(json ,'$.connection.config.hostPort', '$.connection.config.numberOfStatus')
+WHERE serviceType = 'Dagster'
```

Comment:
for this change I believe we might need to delete the current ingestion pipelines as well cc: @pmbrull

Reply:
Hi folks, I don't think so 🤔 in the Airflow APIs we will be picking up the service and its connection dynamically via an API call, no info about the service should be present in the IngestionPipeline object itself.

Reply:
it's the logic from here https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py#L141 Do let me know if I am missing anything, been a while since I touched this hehe